kani-driver/src/concrete_playback/test_generator.rs

```diff
@@ -84,71 +94,83 @@ impl KaniSession {
         &self,
         src_path: &str,
         proof_harness_end_line: usize,
-        unit_test: &UnitTest,
+        unit_tests: Vec<UnitTest>,
     ) -> Result<()> {
-        let unit_test_already_in_src =
-            self.add_test_inplace(src_path, proof_harness_end_line, unit_test)?;
+        // compute range to run rustfmt on.
+        let concrete_playback_num_lines: usize =
+            unit_tests.iter().map(|unit_test| unit_test.code.len()).sum();
+        let unit_test_start_line = proof_harness_end_line + 1;
+        let unit_test_end_line = unit_test_start_line + concrete_playback_num_lines - 1;
```

Comment:
you should probably take `is_new_injection` into consideration here.

Reply:
930efaf7f4edfaa107774da0e770ce8fe252b931